app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+end
```

Comment:
- [ ] Kindly create a private method that updates the post as per requirement.

e.g

```
private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```